libs/rtefsutils/include/RteFsUtils.h

```diff
@@ -248,9 +248,8 @@ class RteFsUtils
    * @brief normalize path by ensuring that it's absolute and canonical
    * @param path folder to be normalized
    * @param base root folder to form absolute
-   * @return true if path is normalized
   */
-  static bool NormalizePath(std::string& path, std::string base = "");
+  static void NormalizePath(std::string& path, std::string base = "");
```

Comment:
since the signature is changed, could we also change 
std::string base = "" 
to
const std::string& = RteUtils::EMPTY_STRING

Reply:
Fixed.